app/controllers/api/v1/mc_questions_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::McQuestionsController < ApplicationController
+    before_action :set_mc_question, only: [:show, :edit, :update, :destroy]
+    skip_before_action :verify_authenticity_token
+
+    # GET /mc_questions or /mc_questions.json
+    def index
+      @mc_questions = McQuestion.all
+      render json: @mc_questions.to_json(include: :mc_options)
```

Comment:
should we remove this since we wont ever have  a case where we want to see all the mcquestions in the database?

Reply:
yeah we can, wasn't sure if we needed this either.